apps/prairielearn/src/sync/fromDisk/assessments.ts

```diff
@@ -133,9 +133,12 @@ function getParamsForAssessment(
           return {
             qid: alternative.id,
             maxPoints: alternative.maxPoints ?? question.maxPoints ?? null,
-            points: alternative.points ?? question.points ?? null,
+            points: (alternative.points ?? question.points ?? null) as number | number[] | null,
```

Comment:
The syncing code was incorrectly typed -- it uses the output types from Zod, but Zod never actually parses the data -- the JSONSchema does, and ajv's validation doesn't e.g. fill in default values. I noticed this when I didn't get a type error on a conditional property access, when I should have gotten one.

Reply:
This has been resolved by #12675. We now parse all incoming data with a Zod schema.